src/librvnpal/src/posix/mapping.c

```diff
@@ -158,3 +158,39 @@ rvn_unmap(void *address, int64_t size, int32_t delete_on_close, int32_t *detaile
 
     return rc;
 }
+
+EXPORT int32_t
+rvn_mmap_dispose_handle(const char *file_path, void *handle, int32_t delete_on_close, int32_t *detailed_error_code)
```

Comment:
Let's not get the `file_path` here, instead, we can use: https://stackoverflow.com/questions/1188757/getting-filename-from-file-descriptor-in-c